src/components/pages/homePage.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Characters from '../characters';
+
+const HomePage = () => (
```

Comment:
- [ ] Once more don't forget that this is a mobile web application. We requested to make sure mobile users can have a good experience while using the app hence app should be responsive and adhere to the design provided in our project's requirements. You can check our the differences in design in the table below and make the necessary changes.

| Design guidelines :+1:  | Your design |
| --- | --- |
| ![Screenshot from 2023-05-26 15-00-43](https://github.com/Kwaku28/Disney-Characters/assets/63087491/0ddfdcc0-7b53-45b4-a3fd-6b127dbc7c42) | ![Screenshot from 2023-05-26 14-58-10](https://github.com/Kwaku28/Disney-Characters/assets/63087491/fe75529e-c535-449e-905a-6497de115bc9) |